src/pages/archives/ArchivePage.js

```diff
@@ -289,7 +290,20 @@ class ArchivePage extends Component {
               {addNewlineInDesc(this.state.item.description)}
             </div>
             <div className="col-lg-6 details-section-metadata">
-              <Citation item={this.state.item} />
+              <SocialButtons
+                buttons={[
+                  "Email",
+                  "Facebook",
+                  "Twitter",
+                  "Pinterest",
+                  "Tumblr",
+                  "Reddit"
+                ]}
```

Comment:
I can not tell from the ticket description https://webapps.es.vt.edu/jira/browse/LIBTD-2167. did anyone already decide we will use these six services?